### PR TITLE
Ying Zeng Homework1

### PR DESCRIPTION
**Pull request by zyzyxzy**
Please check the 8c98176 as my final version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,24 @@
-# hw01
-Repository for homework 1
+# Biography
+
+
+![](https://media.licdn.com/mpr/mpr/shrinknp_400_400/AAEAAQAAAAAAAAqhAAAAJDNlZGE2ZDY3LTZlYTQtNGU1OS1iZTViLThiNjAzOWRlZmI4Yg.jpg)
```
Comment (smixon) [inline README.md:4]: It looks like this should be an image, but the code is not correct. You need to provide a name within the [].

File: README.md [hunk 2]
```diff
@@ -1,2 +1,24 @@
...
+
+## Ying Zeng
+
+https://github.com/zyzyxzy
```
Comment (smixon) [inline README.md:8]: Nice! It took me a minute to figure out this shortcut for including a link.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,24 @@
...
+
+
+### Introduction
+
+Ying Zeng is currently a **MAPSS** student in *psychology* at the University of Chicago. She graduated from **Peking University** in 2017, with dual bachelor degrees in *psychology* and *economics*.
```
Comment (smixon) [inline README.md:13]: Obviously, the use of bold and italics is specifically for the purpose of checking boxes on the assignment, but I like that you were consistent in the use of the two - bold for university and italic for degrees.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,24 @@
...
+
+
+### Research Interests
+
+My research interests covers various topics in social psychology and consumer behavior. Specifically, I am quite interested in:
+
+* Judgement decision making
+* Perspective taking
```
Comment (smixon) [inline README.md:21]: I don't know what 'perspective taking' is, but you've managed to use another required aspect in a coherent context.

**Review by smixon — COMMENTED** [review]
Ying, overall your code and structure looks good! I also feel like I learned some interesting things about you. The assignment was simple but you have managed to include the necessary aspects in a logical order that made the file easy to understand.

**Review by mbokanga — COMMENTED** [review]
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      check     |              |            |
| **Coding strategy**         |  check         |              |            |
| **Presentation**            |     check      |           |            |
| **Achievement, creativity** |      check     |              |            |
| **Ease of access**          |     check      |              |            |

everything looks good

**Comment by cernhofer** [comment]
Evaluation HW-01 for zyzyxzy
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |        x   |              |            |
| **Achievement, creativity** |   x        |              |            |
| **Ease of access**          |           |    x          |            |

Remarks:
* Great job! You included a number of markdown elements and your final product looks very clean and professional.
* In the future, you can include your reflections/progress in the comment of the pull request.

**Pull request closed without merging**